src/app/pages/my-polls/my-polls.component.html

```diff
@@ -23,60 +23,56 @@
   </nz-page-header>
 
   <!-- Poll list -->
-  <ng-scrollbar class="scrollbar">
-    <div class="poll-list-container" *ngIf="polls !== undefined && polls !== null && polls.length > 0">
-      <nz-collapse nzBordered="false" nzAccordion class="no-select">
-        <nz-collapse-panel #p *ngFor="let poll of polls" [nzHeader]="header" [nzExtra]="options"
-                           [nzExpandedIcon]="expandedIcon">
-          <!-- Item content -->
-          <nz-row [nzGutter]="12">
-            <nz-col [nzSpan]="12">
-              <nz-card>
-                <nz-statistic [nzValue]="10" nzTitle="Participants"
-                              [nzValueStyle]="{ color: '#3F8600' }">
-                </nz-statistic>
-              </nz-card>
-            </nz-col>
-            <nz-col [nzSpan]="12">
-              <nz-card>
-                <nz-statistic [nzValue]="5" nzTitle="Questions"
-                              [nzValueStyle]="{ color: '#CF1322' }">
-                </nz-statistic>
-              </nz-card>
-            </nz-col>
-          </nz-row>
+  <div class="poll-list-container" *ngIf="polls !== undefined && polls !== null && polls.length > 0">
```

Comment:
should be the same as `*ngIf="polls?.length"`

Reply:
Good thought. I'll try this